unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PlayerName/PlayerName.cs

```diff
@@ -132,8 +135,11 @@ public void RemoveVisibilityConstaint(string constraint)
     public void SetForceShow(bool forceShow)
     {
         canvas.enabled = forceShow || namesVisible.Get();
-        canvas.sortingOrder = forceShow ? FORCE_CANVAS_SORTING_ORDER : DEFAULT_CANVAS_SORTING_ORDER;
+
+        //canvas.sortingOrder = forceShow ? FORCE_CANVAS_SORTING_ORDER : DEFAULT_CANVAS_SORTING_ORDER;
```

Comment:
Do we want to leave it commented for any reason? or we can remove it?

Reply:
Nope, good catch